inception-kb/src/test/java/de/tudarmstadt/ukp/inception/kb/KnowledgeBaseServiceRemoteTest.java

```diff
@@ -363,25 +369,29 @@ public static KnowledgeBase setOWLSchemaMapping(KnowledgeBase kb) {
         kb.setPropertyTypeIri(RDF.PROPERTY);
         return kb;
     }
-    
+
     private static class TestConfiguration
     {
         private final String url;
         private final KnowledgeBase kb;
         private final String testIdentifier;
-        public TestConfiguration(String aUrl, KnowledgeBase aKb, String atestIdentifier)
+        private final String rootIdentifier;
+
+        public TestConfiguration(String aUrl, KnowledgeBase aKb, String atestIdentifier,
+                String aRootIdentifier)
```

Comment:
Shouldn't this be a set?

Reply:
I don't see a need of Set here as it just checks with root concept.

Reply:
There could be more than one root concept.